### PR TITLE
Migrate all subscriptions from old host to new

### PR DESCRIPTION
**Pull request by asood123**

### CODE REVIEW

File: server/gateways/stripe.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ const getOrCreatePlan = (stripeAccount, plan) => {
 
   return stripeClient.plans.retrieve(plan.id)
     .catch((err) => {
-      if (err.type === 'StripeInvalidRequest' && _.contains(err.message, 'No such plan')) {
+      if (err.type === 'StripeInvalidRequestError' && _.contains(err.message, 'No such plan')) {
```
Comment (xdamman) [inline server/gateways/stripe.js]: I'd add a `console.log` to make sure you log potential errors even if they don't match that specific error that you expect.

File: server/gateways/stripe.js [hunk 2]
```diff
@@ -36,6 +36,30 @@ const createSubscription = (stripeAccount, customerId, subscription) => {
 };
 
 /**
+ * Retrieve stripe subscription
+ */
+const retrieveSubscription = (stripeAccount, customerId, stripeSubsriptionId) => {
+  return client(stripeAccount).customers.retreiveSubscription(customerId, stripeSubsriptionId);
```
Comment (sedubois) [inline server/gateways/stripe.js]: is the typo in `retreiveSubscription` intended?

Comment (asood123) [inline server/gateways/stripe.js]: done

Comment (asood123) [inline server/gateways/stripe.js]: nope. good catch. Ended up not using that code.

File: server/gateways/stripe.js (hunk 1 above)
Comment (asood123) [inline server/gateways/stripe.js]: done.

File: scripts/migrate_subscriptions_in_stripe.js [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+    // store the new stripeSubscription info in our table
+    .then(newStripeSubscription => {
+      const oldData = currentOCSubscription.data;
+
+      return currentOCSubscription.updateAttributes({
+        data: Object.assign({}, newStripeSubscription, { oldData }),
+        stripeSubscriptionId: newStripeSubscription.id
+      });
+    })
+    // delete the old subscription from stripe
```
Comment (xdamman) [inline scripts/migrate_subscriptions_in_stripe.js]: Before deleting, I'd add a `catch` and make sure there is no error before.

Comment (asood123) [inline scripts/migrate_subscriptions_in_stripe.js]: If there is an error, won't it jump to the catch on line 98?

**Pull request merged**